### ISSUE TITLE
[modloader.mod] Error messages of libopenmpt on "TryLoadSound()" with other formats

### ISSUE THREAD

**Issue by GWRon** [bmx-ng/audio.mod#18]
Sample code see #16 and #15 

When importing "audio.modloader" it registers `TSoloudModLoader`. If you now load an audio file (like a ".wav" file) this TSoloudModLoader instance will try to load it. And as it fails doing so it prints out an error message.

```
Type TSoloudModLoader Extends TAudioSourceLoader

	Method LoadAudioSource:TSLLoadableAudioSource( stream:TStream, flags:Int ) Override
		Local sound:TSLLoadableAudioSource = New TSLOpenmpt
		If sound.loadStream(stream) = SO_NO_ERROR Then
			Return sound
		End If
		sound.destroy()
	End Method
	
End Type


Type TSLOpenmpt Extends TSLLoadableAudioSource
...
	Method loadStream:Int(stream:TStream) Override
		Local sf:TStreamFile = New TStreamFile.Create(stream)
		Return Openmpt_LoadFile(asPtr, sf.filePtr)
	End Method
```

So it calls "Openmpt_LoadFile()" which is in soloud_c_openmpt.c:
```C
int Openmpt_loadFile(void * aClassPtr, File * aFile)
{
	Openmpt * cl = (Openmpt *)aClassPtr;
	return cl->loadFile(aFile);
}
```
This is leading to "soloud.mod/soloud/src/audiosource/openmpt/soloud_openmpt.cpp"
```C
extern "C"
{
	void * openmpt_module_create_from_memory(const void * filedata, size_t filesize, void *logfunc, void * user,void * ctls);
```

This is leading to "libopenmpt.mod/openmpt/libopenmpt/libopenmpt_c.cpp":
```C
openmpt_module * openmpt_module_create_from_memory( const void * filedata, size_t filesize, openmpt_log_func logfunc, void * user, const openmpt_module_initial_ctl * ctls ) {
	return openmpt_module_create_from_memory2( filedata, filesize, logfunc, user, NULL, NULL, NULL, NULL, ctls );
}

openmpt_module * openmpt_module_create_from_memory2( const void * filedata, size_t filesize, openmpt_log_func logfunc, void * loguser, openmpt_error_func errfunc, void * erruser, int * error, const char * * error_message, const openmpt_module_initial_ctl * ctls ) {
	try {
		openmpt_module * mod = (openmpt_module*)std::calloc( 1, sizeof( openmpt_module ) );
		if ( !mod ) {
			throw std::bad_alloc();
		}
		std::memset( mod, 0, sizeof( openmpt_module ) );
		mod->logfunc = logfunc ? logfunc : openmpt_log_func_default;
		mod->loguser = loguser;
...
```

So there is a log function which could be set.

The error itself is generated in libopenmpt.mod/openmpt/libopenmpt/libopenmpt_impl.cpp:
```C
void module_impl::load( const OpenMPT::FileCursor & file, const std::map< std::string, std::string > & ctls ) {
	loader_log loaderlog;
	m_sndFile->SetCustomLog( &loaderlog );
	{
		int load_flags = OpenMPT::CSoundFile::loadCompleteModule;
		if ( m_ctl_load_skip_samples ) {
			load_flags &= ~OpenMPT::CSoundFile::loadSampleData;
		}
		if ( m_ctl_load_skip_patterns ) {
			load_flags &= ~OpenMPT::CSoundFile::loadPatternData;
		}
		if ( m_ctl_load_skip_plugins ) {
			load_flags &= ~(OpenMPT::CSoundFile::loadPluginData | OpenMPT::CSoundFile::loadPluginInstance);
		}
		if ( !m_sndFile->Create( file, static_cast<OpenMPT::CSoundFile::ModLoadingFlags>( load_flags ) ) ) {
			throw openmpt::exception("error loading file");
		}
```

TL/DR: it would be nice if this kind of error is hidden from the users - maybe it should only be printed if _all_ loaders failed to load the audio file.
